mobify/platform-scaffold
# Add Favicon

**JIRA**: [WEB-1289](https://mobify.atlassian.net/browse/WEB-1289)

## Changes
- Added Favicon

## Impact

A favicon was missing from our PWA, and Web Page Test seems to indicate that the browser will still attempt to download a default favicon from the domain at `https://www.merlinspotions.com/favicon.ico`, but since that doesn't exist, it just 404s. The browser seems to block and further downloads from happening (see screenshot below).

I assert (assuming, really) that by adding our favicon, the browser will become unblocked entirely, or at least less blocked by downloading a rather small asset (1.4kb) faster than hanging on a 404 asset.

![screen shot 2017-03-14 at 5 19 04 pm](https://cloud.githubusercontent.com/assets/734535/23928149/70cc6b30-08db-11e7-9a68-1d057de6bfa4.png)

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Verify that the favicon loads in the network tab of Chrome's inspector
